### DISCUSSION TITLE
Clicking on the icon from the taskbar to call out the software is slow

### DISCUSSION THREAD

**Discussion by youhers** [rustdesk/rustdesk#8908] [General]
Clicking on the desktop icon to open the software quickly - what's going on? It's very slow and takes a lot of time when working.

**Comment by rustdesk** [comment]
@21pages  slow?

**Reply by 21pages** [reply]
Can't reproduce, @youhers  could you record a video?

**Comment by youhers** [comment]
No problem

**Comment by youhers** [comment]
It seems that I can't send a video file, can I send it to you by email?

**Reply by 21pages** [reply]
Yes,  `c3VuYm9lYXN5QGdtYWlsLmNvbQo=`, base64

**Reply by youhers** [reply]
Is this normal?

**Reply by 21pages** [reply]
https://github.com/rustdesk/rustdesk/blob/a12969be307909d3540694e5d07b424ae2b97d95/src/tray.rs#L165

Detect left mouse button click and then launch the application, one of these two is slow on your computer.

**Comment by youhers** [comment]
![动画](https://github.com/user-attachments/assets/8073eb65-0709-45da-947f-68a1b3300a9a)
